source/administration/config-settings.rst

```diff
@@ -3269,6 +3268,52 @@ Path and filename of the license file on disk. On startup, if Mattermost cannot
 | This feature's ``config.json`` setting is ``"LicenseFileLocation": ""`` with string input.  |
 +---------------------------------------------------------------------------------------------+
 
+TLS Minimum Version
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The minimum TLS version used by the Mattermost server. TLS v1.2 is default given insecurities for TLS 1.0 and 1.1.
+
+This setting only takes effect if you are using the built-in server binary directly, and not using a reverse proxy layer such as NGINX.
+
++-------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"TLSMinVer": "1.2"`` with string input. |
++-------------------------------------------------------------------------------------+
+
+Allow Secure TLS Transports Only
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: Only allow secure TLS transport, forcing the browser to request all resources via HTTPS. Learn more `here <https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security>`_.
+
+**False**: No restrictions on TLS transport. Resources can be requested via HTTP or HTTPs.
+
```

Comment:
```Resources can be``` ==> ```Resources are````.  0/5 as I am not sure if this is accurate - the current wording makes me wonder if there is another setting for HTTP vs HTTPs, which if there is, should we link?